src/components/shared/IbanFields.vue

```diff
@@ -17,9 +17,9 @@
 			<ScrollTarget target-id="iban-calculator-scroll-target"/>
 
 			<div class="iban-calculator-content">
-				<h2 class="icon-title">
+				<h3 class="icon-title">
```

Comment:
the bank datasection has a h2 heading.
the iban calculator is a nested component inside of the bank data section.
So the iban calculator should have a h3 headline, no?

Reply:
>the bank datasection has a h2 heading.
Each section should have h2 -> h3 -> h4 etc, because the other boxes are separate they shoudl be h2. It should be similar to how you do headings in a Word document.

>So the iban calculator should have a h3 headline, no?
Yep you're right, I missed that